beacon-chain/sync/service.go

```diff
@@ -126,6 +130,37 @@ func (ss *Service) ReceiveBlockHash(data *pb.BeaconBlockHashAnnounce, peer p2p.P
 	return nil
 }
 
+// verifyAttestation verifies incoming attestation with the block, when a block is produced,
```

Comment:
`verifyAttestation verifies the proposer's attestation of the block. Proposers broadcast the attestation along with the block to its peers.`